src/components/molecules/GameCard/GameCard.module.scss

```diff
@@ -2,45 +2,90 @@
 @use 'src/styles/placeholders';
 
 .gameCard {
-    $width: 325px;
-    $height: 400px;
+    $width: 275px;
+    $height: 475px;
     @extend %flex-column;
     gap: 0.5rem;
     padding: 0.5rem;
+    border-radius: 0.5rem;
     width: $width;
     max-width: $width;
     height: $height;
     max-height: $height;
-    color: vars.$text-light;
-    background: vars.$bg-dark;
+    background: vars.$bg-dark-lightest;
 }
 
-.gameCardCoverImg {
-    $height: 200px;
-    width: 100%;
+.gameImgContainer {
+    $height: 250px; // 187px half image height
+    $width: 175px; // 132px half image width
+    width: $width;
+    max-width: $width;
     height: $height;
     max-height: $height;
-    object-fit: fill;
+
+    img {
+        border-radius: 5px;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        overflow: hidden;
+    }
 }
 
 .gameCardHead {
     @extend %flex-row;
     align-items: center;
     gap: 0.5rem;
+}
 
-    div {
-        @extend %flex-column;
+.headText {
+    @extend %flex-column;
 
-        h2 {
-            $width: 250px;
-            @extend %text-ellipsis;
-            font-size: 1.5rem;
-            width: $width;
-            max-width: $width;
-        }
+    width: calc(
+        100% - 1rem - 50px
+    ); // (ParentWidth - padding&sidebar - Rating component size)
```

Comment:
> width: calc(100% - 1rem - 50px); // (ParentWidth - padding&sidebar - Rating component size)

For clarity, I'd say to reference these values by a variable.

For example, if 50px is the size of the Rating component, use the size variable from that scss sheet here instead of 50px. That way if we ever change the size of the Rating component's styling, it will update here as well.  Same with padding&sidebar, create variables for those wherever their styling exists, and use them here.

Ultimately it should look something like the following:
```
// Don't use these variable names, use whatever names they already have
// But hopefully you get my point
// By using good descriptive variable names you won't have to leave behind
// a comment explaining what each of these values are for
width: calc(100% - $padding-sidebar-size - $ratingSize);
```